packages-next/auth/src/gql/getPasswordResetSchema.ts

```diff
@@ -59,7 +61,7 @@ export function getPasswordResetSchema({
     `,
     resolvers: {
       Mutation: {
-        async [gqlNames.sendItemPasswordResetLink](root: any, args: any, context: any) {
+        async [gqlNames.sendItemPasswordResetLink](root, args, context) {
```

Comment:
Do we not want to add a `KeystoneContext` on here (and other places in the auth package)?